### DISCUSSION TITLE
Team Requirements Architekt

### DISCUSSION THREAD

**Discussion by tobizwetz2** [pcg-platinus/AM-Uebung#7] [General]
Stakeholder Analyse

**Comment by tobizwetz2** [comment]
- Familie
-   Hausherr
-   Frau
-   Oma Opa
-   (implizit: Kinder?)


- Solar-Firma
- Heizungshersteller
- Stromanbieter
- Internetanbieter
- ELTAKO (Stromzähler)
- App-Entwickler
- Smart Home Anbieter
- Elektriker
- Anbieter der Sensoren (HomeMatic)
- Sicherheitsdienst Anbieter
- Schlüsselanbierer 
- Installateur


- Freunde
- Gäste
- Besucher


- Versicherungsmakler
- Bauvorschriften
- Gesetzgeber (Kameraüberwachung)
- Brandschutzvorschriften


- Mieter
- Hausverwaltung
- Reinigungskraft
- Hausherr

**Comment by MBpraesents** [comment]
![A5AB1677-0AD4-4B98-ABA3-8B9BE6838C7E](https://user-images.githubusercontent.com/125259668/218693849-3ba0d536-9dc5-4ca0-9825-0c14ab106a37.jpeg)

**Comment by AlexM5452** [comment]
![WIN_20230215_09_25_00_Pro](https://user-images.githubusercontent.com/125259234/218972856-5a46319c-a470-4501-bd54-38e3428f5b37.jpg)
